### PR TITLE
Updated default ryuk image to testcontainers/ryuk:0.3.3.

### PR DESCRIPTION
**Pull request by silh**
Updated default reaper image to a newer version.
Old one didn't have arm version, this one does.

### CODE REVIEW

**Review by mdelapenya — APPROVED** [review]

**Comment by mdelapenya** [comment]
Thanks for adding this! Waiting for the CI for final review 👋

**Comment by gianarb** [comment]
Thanks a lot!

**Pull request merged**
